src/main/kotlin/me/serce/solidity/lang/types/internal.kt

```diff
@@ -88,21 +144,76 @@ class SolInternalTypeFactory(project: Project) {
           $blockType block;
           $msgType msg;
           $txType tx;
-          uint now;
```

Comment:
I was a bit confused by this change, but then I saw 👍 
<img width="731" alt="Screenshot 2023-03-29 at 10 27 19 pm" src="https://user-images.githubusercontent.com/1780970/228521689-f2becad1-f62f-4a3e-a921-303e246d7313.png">